backend/src/test/java/com/woowacourse/naepyeon/service/TeamServiceTest.java

```diff
@@ -126,14 +145,43 @@ void delete() {
     @DisplayName("회원을 모임에 가입시킨다.")
     void joinMember() {
         final String nickname = "닉네임";
-        final Long joinedId = teamService.joinMember(team.getId(), member.getId(), nickname);
+        final Long joinedId = teamService.joinMember(team1.getId(), member.getId(), nickname);
         final TeamParticipation findTeamParticipation = teamMemberRepository.findById(joinedId)
                 .orElseThrow();
 
         assertAll(
                 () -> assertThat(findTeamParticipation.getMember().getId()).isEqualTo(member.getId()),
-                () -> assertThat(findTeamParticipation.getTeam().getId()).isEqualTo(team.getId()),
+                () -> assertThat(findTeamParticipation.getTeam().getId()).isEqualTo(team1.getId()),
                 () -> assertThat(findTeamParticipation.getNickname()).isEqualTo(nickname)
         );
     }
+
+    @Test
+    @DisplayName("모든 모임을 조회한다.")
+    void findAll() {
+        final TeamsResponseDto teams = teamService.findAll();
+        final List<String> teamNames = teams.getTeams()
+                .stream()
+                .map(TeamResponseDto::getName)
+                .collect(Collectors.toList());
+
+        assertThat(teamNames).contains(team1.getName(), team2.getName(), team3.getName());
```

Comment:
```suggestion
        assertThat(teamNames).containsExactly(team1.getName(), team2.getName(), team3.getName());
```
현재 contains으로 테스트코드를 작성하셨는데 이럴 경우 검증해야 할 값이 한개가 누락되어도 성공하는 문제가 있을 것 같은데 `containsExactly`로 바꿔보시는건 어떤가요?

Reply:
containsExactly는 순서까지 맞추는 걸로 알고 있고 contains는 값이 있는 지만 검사하는 걸로 알고 있습니다!
해당 부분은 db에서 조회된 결과를 stream으로 name만 뽑아낸 리스트를 검증하는 것이기에 순서까지 보장시키려면 db에서 순서까지 보장하도록 쿼리를 작성하고 스트림에서도 다른 코드를 짜야 할텐데 비즈니스 요구사항과는 거리가 멀어보여서 contains를 사용했어요.